content/tutorial/tutorial.md

```diff
@@ -472,19 +472,19 @@ class Square extends React.Component {
 }
 ```
 
-When a Square is clicked, the `onClick` function provided by the Board is called. Here's a review of how this is achieved:
+Gdy użytkownik kliknie na pole, zostanie wywołana funkcja `onClick` dostarczona przez komponent `Board`. Dzieje się to dlatego, że:
 
-1. The `onClick` prop on the built-in DOM `<button>` component tells React to set up a click event listener.
-2. When the button is clicked, React will call the `onClick` event handler that is defined in Square's `render()` method.
-3. This event handler calls `this.props.onClick()`. The Square's `onClick` prop was specified by the Board.
-4. Since the Board passed `onClick={() => this.handleClick(i)}` to Square, the Square calls `this.handleClick(i)` when clicked.
-5. We have not defined the `handleClick()` method yet, so our code crashes.
+1. Atrybut `onClick` wbudowanego komponentu DOM `<button>` informuje Reacta, żeby ustawił obserwator (ang. *listener*) na kliknięcie.
```

Comment:
Jeśli "obserwator kliknięć" nie jest przyjętym powszechnie tłumaczeniem "event listener" to propnuję oddać to bardziej opisowo lub pójść w stronę "słuchacza".

```suggestion
1. Atrybut `onClick` wbudowanego komponentu DOM `<button>` informuje Reacta, zaczął nasłuchiwać kliknięć użytkownika.
```